packages/store-indexer/bin/postgres-decoded-indexer.ts

```diff
@@ -35,16 +36,21 @@ const publicClient = createPublicClient({
 });
 
 const chainId = await publicClient.getChainId();
-const database = drizzle(postgres(env.DATABASE_URL, { prepare: false }));
+const readDatabase = drizzle(postgres(env.READ_DATABASE_URL, { prepare: false }));
+const writeDatabase = drizzle(postgres(env.WRITE_DATABASE_URL, { prepare: false }));
 
-const { storageAdapter, tables } = await createStorageAdapter({ database, publicClient });
+const { storageAdapter, tables } = await createStorageAdapter({
+  readDatabase: readDatabase,
+  writeDatabase: writeDatabase,
```

Comment:
can we clarify the naming here? maybe `bytesDatabase` and `decodedDatabase`? (would love to come up with some good naming for these DB types because we throw around a lot of different names)

and similar treatment above when we initiate but maybe allow for `DATABASE_URL` to be used as a fallback for both (which would give us backwards compat)

Reply:
I like the idea of having the fallback, perhaps if WRITE_DATABASE isn't defined? we do risk trying to solve for every use case though